docs/source/optimizations/hilbert/hilbert_compile_css.ipynb

```diff
@@ -158,6 +158,40 @@
     "print(compiler_output.circuit)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a865f7b3",
+   "metadata": {},
+   "source": [
+    "If you would like to compile (or submit) on a different number of qubits, this can be done via the ```grid_shape``` option. This simply sets the shape of the rectangular qubit grid. However, specifying a grid that is incompatible with Hilbert's current capabilities will result in an error when submitting. "
```

Comment:
μ-nit
```suggestion
    "If you would like to compile (or submit) on a different number of qubits, this can be done via the ```grid_shape``` option. This simply sets the shape of the rectangular qubit grid. However, specifying a grid that is incompatible with Hilbert's current capabilities will result in an error when submitting."
```